website/content/docs/secrets/kmip-profiles.mdx

```diff
@@ -0,0 +1,269 @@
+---
+layout: docs
+page_title: KMIP - Profiles Support
+description: |-
+  The KMIP profiles define the use of KMIP objects, attributes, operations, message elements
+  and authentication methods within specific contexts of KMIP server and client interaction.
+  These profiles define a set of normative constraints for employing KMIP within a particular
+  environment or context of use.
+---
+
+## [Baseline Server][baseline-server]
+1. <details>
+  <summary>Supports the following objects</summary>
+
+    | Object                                                                    | Supported |
+    | ------------------------------------------------------------------------- | :-------: |
+    | Attribute [[KMIP-SPEC] 2.1.1][kmip-spec-2.1.1]                            | ✅        |
```

Comment:
What about making it so that the links appear as `[KMIP-SPEC 2.1.1]` rather than `[KMIP-SPEC] 2.1.1`?

Reply:
Done.